theme/image.php

```diff
@@ -78,7 +78,7 @@
 									printf( __( 'Comments are closed, but you can leave a trackback: <a class="trackback-link" href="%s" title="Trackback URL for your post" rel="trackback">Trackback URL</a>.', 'braces' ), esc_url( get_trackback_url() ) );
 								}
 								elseif ( comments_open() && ! pings_open() ) { // Only comments open
-									 _e( 'Trackbacks are closed, but you can <a class="comment-link" href="#respond" title="Post a comment">post a comment</a>.', 'braces' );
+									esc_html_e( 'Trackbacks are closed, but you can <a class="comment-link" href="#respond" title="Post a comment">post a comment</a>.', 'braces' );
```

Comment:
I believe this will cause the HTML tags in this string to be escaped, and so the link will be broken if you call an esc_html-related function here. More info: https://codex.wordpress.org/Function_Reference/esc_html .. Ignore me if I'm wrong since I didn't test this myself yet :-)